eq-author/src/components/CommentsPanel/index.js

```diff
@@ -83,10 +107,77 @@ const NameWrapper = styled.div`
   flex-grow: 2;
 `;
 
+const DateWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: start;
+  flex-grow: 2;
+`;
+
+const DeleteComment = styled(DeleteButton)`
+  color: ${colors.grey};
+  font-size: 2.2em;
+  display: ${props => (props.hideDeleteBtn ? "none" : "block")};
+`;
+
+const ButtonGroupStyled = styled(ButtonGroup)`
+  margin-left: auto;
+`;
+
+const EditButtonIcon = styled.button`
+  display: inline-flex;
+  flex: 0 0 auto;
+  color: ${colors.grey};
+  padding: 1em;
+  border-radius: ${radius};
+  font-size: 1em;
+  font-weight: 600;
+  cursor: pointer;
+  line-height: 1;
+  justify-content: center;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+  text-decoration: none;
+  transition: all 100ms ease-out;
+  letter-spacing: 0;
+  position: relative;
+  overflow: hidden;
+  border: 1px solid var(--color-bg);
+  background: url(${IconEdit}) no-repeat center center;
+  height: 25px;
+  width: 25px;
+
+  display: ${props => (props.hideEditBtn ? "none" : "block")};
```

Comment:
You've got two display properties in here where you should only have one.

Reply:
whoops - ta